src/tests/CheckingCards.test.js

```diff
@@ -34,12 +37,24 @@ const accountsZero = [];
 
 describe('CheckingCardsAccount tests', () => {
     it('Render checking cards', () => {
-        const component = render(<CheckingCards accounts={accounts}/>);
+        const component = render(
```

Comment:
As suggested by `eslint`, you should name these constants "view" instead of "component".

Reply:
I remember we saw it in a previous PR. Thanks for reminding me